### PR TITLE
added test e2e olm and addon-operator installation commands

### PR DESCRIPTION
**Pull request by sandhyadalavi**
Added olm and addon-operator installation steps for e2e-tests to run

### CODE REVIEW

**Comment by erdii** [comment]
I don't think it worked :(

> https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/23602/rehearse-23602-pull-ci-openshift-addon-operator-main-e2e/1469043089900310528/artifacts/e2e/test/build-log.txt
k

> === RUN   TestIntegration
>     helpers.go:179: api mock is enabled: loading manifests
>     helpers.go:183: webhook server is enabled: loading manifests
>     setup_test.go:44: created:  /v1, Kind=Namespace /addon-operator
>     setup_test.go:44: created:  apiextensions.k8s.io/v1, Kind=CustomResourceDefinition /addoninstances.addons.managed.openshift.io
>     setup_test.go:44: created:  apiextensions.k8s.io/v1, Kind=CustomResourceDefinition /addonoperators.addons.managed.openshift.io
>     setup_test.go:44: created:  apiextensions.k8s.io/v1, Kind=CustomResourceDefinition /addons.addons.managed.openshift.io
>     setup_test.go:44: created:  apps/v1, Kind=Deployment addon-operator/addon-operator
>     setup_test.go:44: created:  /v1, Kind=ServiceAccount addon-operator/addon-operator
>     setup_test.go:44: created:  rbac.authorization.k8s.io/v1, Kind=Role addon-operator/addon-operator
>     setup_test.go:44: created:  rbac.authorization.k8s.io/v1, Kind=RoleBinding addon-operator/addon-operator
>     setup_test.go:44: created:  rbac.authorization.k8s.io/v1, Kind=ClusterRole /addon-operator
>     setup_test.go:44: created:  rbac.authorization.k8s.io/v1, Kind=ClusterRoleBinding /addon-operator
>     setup_test.go:44: created:  /v1, Kind=Namespace /api-mock
>     setup_test.go:44: created:  /v1, Kind=Service api-mock/api-mock
>     setup_test.go:44: created:  /v1, Kind=Secret api-mock/api-mock
>     setup_test.go:44: created:  apps/v1, Kind=Deployment api-mock/api-mock
>     setup_test.go:44: created:  /v1, Kind=Secret addon-operator/webhook-server-cert
>     setup_test.go:44: created:  apps/v1, Kind=Deployment addon-operator/addon-operator-webhook
>     setup_test.go:44: created:  /v1, Kind=Service addon-operator/webhook-service

File: ci-operator/config/openshift/addon-operator/openshift-addon-operator-main.yaml [hunk 1]
```diff
@@ -59,25 +68,34 @@ tests:
...
+      OO_INDEX: ci-index
+    env:
+      OO_CHANNEL: stable
+      OO_INSTALL_NAMESPACE: '!create'
+      OO_PACKAGE: addon-operator
+      OO_TARGET_NAMESPACES: '!all'
     test:
     - as: test
-      commands: ENABLE_WEBHOOK=true make test-e2e
+      commands: kubectl get csv,subscription,catalogsource -n addon-operator;ENABLE_WEBHOOK=true make test-e2e
```
Comment (mayankshah1607) [inline ci-operator/config/openshift/addon-operator/openshift-addon-operator-main.yaml:83]: Can you run `make ci-operator-config` and commit the changes?

**Review by thetechnick — APPROVED** [review]

**Pull request merged**